app/(dashboard)/mailboxes/[mailbox_slug]/settings/chat/chatWidgetSetting.tsx

```
@@ -1,711 +1,53 @@
-import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/components/ui/select";
-import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
-import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@/components/ui/tooltip";
-import { useDebouncedCallback } from "@/components/useDebouncedCallback";
-import { useOnChange } from "@/components/useOnChange";
 import { mailboxes } from "@/db/schema";
 import { RouterOutputs } from "@/trpc";
-import { api } from "@/trpc/react";
-import SectionWrapper, { SwitchSectionWrapper } from "../sectionWrapper";
-import CodeBlock from "./codeBlock";
-import WidgetHMACSecret from "./widgetHMACSecret";
+import { WidgetInstallation } from "./components/WidgetInstallation";
```

ESLint

[error] Replace `WidgetInstallation·}·from·"./components/WidgetInstallation` with `AutoRespondSettings·}·from·"./components/AutoRespondSettings`

(prettier/prettier)